configs/yolov5/yolov5_s_p6-v62_syncbn_fast_8xb16-300e_coco.py

```diff
@@ -0,0 +1,89 @@
+_base_ = 'yolov5_s-v61_syncbn_fast_8xb16-300e_coco.py'
+
```

Comment:
I think the filename should be `yolov5_s-p6-v62_syncbn_fast_8xb16-300e_coco.py` because p6 is related to the model structure.

Reply:
OK